### DISCUSSION TITLE
Add support for iOS provideAppNotificationSettings

### DISCUSSION THREAD

**Discussion by BarryCarlyon** [firebase/flutterfire#8552] [Feature request]
This is supported in the react native "version" of this library.

So rather than rehash the issue I'll just link to theirs for easy reference

https://github.com/invertase/react-native-firebase/pull/5972

Please add support for `providesAppNotificationSettings` as documented on https://developer.apple.com/documentation/usernotifications/unnotificationsettings/2990404-providesappnotificationsettings?language=objc

So that we can have the button in iOS Notifications settings for our apps and easily open a relevant view in our apps.

**Comment by nilsabdi** [comment]
This would be a great addition!
